packages/node_modules/@webex/plugin-meetings/test/unit/spec/sparkconfig.js

```diff
@@ -0,0 +1,72 @@
+const SCOPE = [
+  'webexsquare:get_conversation',
+  'Identity:SCIM',
+  'spark:kms',
+  'spark:people_read',
+  'spark:rooms_read',
+  'spark:rooms_write',
+  'spark:memberships_read',
+  'spark:memberships_write',
+  'spark:messages_read',
+  'spark:messages_write'
+].join(' ');
+
+const Config = {
+  /**
+     * Returns the operating system based on appVersion.
```

Comment:
Please reformat so it's clear which element needs this header.

Reply:
Not used so removed it

Reply:
@mattnorris  It didnt throw an error for xdescribe . But there was an error for .only